.devcontainer/devcontainer.json

```diff
@@ -27,7 +28,8 @@
   ],
   "remoteUser": "vscode",
   "containerEnv": {
-    "DOCKER_BUILDKIT": "1"
+    "DOCKER_BUILDKIT": "1",
+    "DOCKER_DEFAULT_PLATFORM": "amd64"
```

Comment:
can this somehow be joined as a single value some that we won't have it defined 3 times in different places?
also, can it be "linux/amd64"?

Reply:
I was referring to the values used for all params to be 1. the same, 2. come from the same root value

Can you have another run at this PR and make sure what is the min that works for you? 
Then we can look into how things can be consolidated so that we don't use the same hard-coded value multiple times.

Reply:
I've removed the run-arg and it seems to work ok – will let @jjgriff93 confirm

I'm not sure they should be the same, given that one is the platform for the devcontainer and the other is the platform of any image built inside the devcontainer. Are there any windows/amd64 image built?

Reply:
That feels like unlikely, but happy to add a `-e` if you think it's worth it?